.env.example

```
@@ -42,6 +42,15 @@ UPSTASH_REDIS_REST_URL=https://<database-name>.upstash.io
 # Local dev — copy to packages/web/.env.local or pass via root --env-file:
 NEXT_PUBLIC_API_URL=http://localhost:8789
 NEXT_PUBLIC_APP_URL=http://localhost:3000
-
-# Production Worker: NEXT_PUBLIC_* plain vars must match what you pass into `opennextjs-cloudflare build`
-# (see GitHub Actions vars NEXT_PUBLIC_API_URL / NEXT_PUBLIC_APP_URL in .github/workflows/web.yml).
+C15T_DATABASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The C15T_DATABASE_URL key should go before the NEXT_PUBLIC_API_URL key

(UnorderedKey)